### PR TITLE
add KuCoin Community Chain(KCC) testnet

### PR DESCRIPTION
**Pull request by lingyun92**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
The exporers section is invalid JSON
Also the exporer does not adhere EIP3091 (block is broken - https://scan-testnet.kucoin.one/block/32735)
So they either need to change this or create their own standard

**Comment by lingyun92** [comment]
> The exporers section is invalid JSON
> Also the exporer does not adhere EIP3091 (block is broken - https://scan-testnet.kucoin.one/block/32735)
> So they either need to change this or create their own standard

fixed, thanks.

**Comment by ligi** [comment]
Great! Seems the explorer is close to following the standard now - only tokens is still wrong as far as I see.
https://scan-testnet.kucoin.one/token/0xb296bab2ed122a85977423b602ddf3527582a3da

Also please add the  "standard": "EIP3091" then - after I can merge

**Comment by lingyun92** [comment]
> Great! Seems the explorer is close to following the standard now - only tokens is still wrong as far as I see.
> https://scan-testnet.kucoin.one/token/0xb296bab2ed122a85977423b602ddf3527582a3da
> 
> Also please add the "standard": "EIP3091" then - after I can merge

works now

File: _data/chains/eip155-322.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "KuCoin Community Chain Testnet",
+  "chain": "KCC",
+  "network": "testnet",
+  "rpc": [
+    "https://rpc-testnet.kucoin.one",
```
Comment (ligi) [inline _data/chains/eip155-322.json:6]:
```suggestion
    "https://rpc-testnet.kucoin.one"
```

**Review by ligi — CHANGES_REQUESTED** [review]
CI found this extra comma and complained about it

**Review by ligi — APPROVED** [review]

**Comment by ligi** [comment]
Long time in the making - but finally merged ;-)

**Pull request merged**